guides/tutorial/part-2/ember-data.md

```diff
@@ -210,17 +209,17 @@ module('Unit | Model | rental', function (hooks) {
 });
 ```
 
-This model test is also known as a _[unit test](../../../testing/testing-models/)_. Unlike any of the other tests that we've written thus far, this test doesn't actually _render_ anything. It just instantiates the rental model object and tests the model object directly, manipulating its attributes and asserting their value.
+Ce test de modèle est un [test unitaire (_unit test_)](../../../testing/testing-models/). Contrairement aux tests que nous avons écrit jusqu'à maintenant, celui-ci "n'affiche" rien. Il ne fait qu'instancier le modèle `rental` et le tester directement, en manipulant ses attributs et en faisant des assertions sur leur valeur.
 
-It is worth pointing out that Ember Data provides a `store` _[service](../../../services/)_, also known as the Ember Data store. In our test, we used the `this.owner.lookup('service:store')` API to get access to the Ember Data store. The store provides a `createRecord` method to instantiate our model object for us.
+Il est important de noter qu'Ember Data fournit un [service `store`](../../../services/) (magasin, réserve), aussi connu sous le nom de _Ember Data store_. Dans notre test, l'API `this.owner.lookup('service:store')` accède au _store_ d'Ember Data. Le _store_ fournit une méthode `createRecord` (créer un enregistrement) qui instancie un modèle pour nous.
 
-Running the tests in the browser confirms that everything is working as intended:
+Exécuter les tests dans le navigateur confirme que tout fonctionne comme prévu&nbsp;:
 
-<img src="/images/tutorial/part-2/ember-data/pass-1@2x.png" alt="All the tests pass!" width="1024" height="1024">
+<img src="/images/tutorial/part-2/ember-data/pass-1@2x.png" alt="Tous les tests passent !" width="1024" height="1024">
 
-## Loading Models in Routes
+## Charger un modèle dans une route
 
-Alright, now that we have our model set up, it's time to refactor our route handlers to use Ember Data and remove the duplication!
+Très bien, maintenant que notre modèle est prêt, il est temps de refactorer nos gestionnaires de route pour utiliser Ember Data et supprimer la duplication de code&nbsp;!
```

Comment:
Always tricky this one, on l'utilise à toutes les sauces, what about this:

```suggestion
Très bien, maintenant que notre modèle est prêt, il est temps de _refactor_ nos gestionnaires de route pour utiliser Ember Data et supprimer le code dupliqué&nbsp;!
```

Reply:
I wouldn't change "refactorer" here, it's pretty common in French-speaking environments.